### PR TITLE
Add check service status

### PR DESCRIPTION
**Pull request by calvn**
Reviving dead PR #906

### CODE REVIEW

File: plugins/system/check-service-status.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+#
+# Check service status
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
```
Comment (tas50) [inline plugins/system/check-service-status.rb]: This can be removed since we don't support Ruby 1.8.7 anymore

**Comment by cleung2010** [comment]
@tas50 somehow a rabbitmq check is causing the build to fail even though that file was not touched, any clues?

https://travis-ci.org/sensu/sensu-community-plugins/builds/64917469

**Comment by cleung2010** [comment]
@mattyjones Can this PR be merged? It would be great that if the plugin can provide service checks :)

**Comment by mattyjones** [comment]
@cleung2010  Can you remove the `;` per rubocop please and then I will merge.

Thanks!

**Comment by cleung2010** [comment]
I didn't change the rabbitmq plugin, but I'll remove gladly remove it. Just wanted to point out that the `;` on `check-rabbitmq-messages.rb` was somehow merged to master https://github.com/sensu/sensu-community-plugins/blob/master/plugins/rabbitmq/check-rabbitmq-messages.rb

**Pull request closed without merging**